ts/components/AboutView.tsx

```diff
@@ -40,7 +40,7 @@ export const AboutView = () => {
         theme: window.theme,
       });
     }
-  }, [window.theme]);
```

Comment:
@Bilb Why has this been removed? If someone changes the theme while the about window is open we want it to update?

Reply:
`window.theme` is not part of redux, a props or a state.
If someone changes theme while the about window is opened, the theme won't change in the about window. It needs to be opened and closed to be updated

Reply:
I'm like 90% sure I had it updating when people changed the theme though? 🤔 Maybe I remember incorrectly

Reply:
Confirmed not related to the PR will add to the backlog.